chaincode-labs/chaincode-podcast/the-bitcoin-development-kit-bdk.md

```diff
@@ -112,15 +114,15 @@ And he really did like a lot of contribution.
 He was using it for, I don't remember his name, GAN, I think, something like that.
```

Comment:
I couldn't avoid to think this `GAN` transcription was wrong. Luckily, I stumbled on the actual project a couple of days ago: https://github.com/GoUpNumber/gun. Probably the inclusion of hyperlinks goes beyond the objectives of transcription, but at least we can change it to \`gun\` , WDYT @kouloumos ?

Reply:
Of course, anybody can open a PR to make a change (small or big) to a transcript.